configs/_base_/datasets/coco_detection.py

```diff
@@ -8,17 +8,16 @@
 #         './data/': 's3://openmmlab/datasets/detection/',
 #         'data/': 's3://openmmlab/datasets/detection/'
 #     }))
-file_client_args = dict(backend='disk')
 
 train_pipeline = [
-    dict(type='LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='LoadImageFromFile'),
```

Comment:
seems `backend_args=xxx` is necessary?

Reply:
yes.
In mmcv2.0.0rc4 and mmengine>0.2.0, 
file_client_args rename backend_args and  dict(backend='disk') rename dict(backend='local')
The default value for backend_args is None, the file backend is inferred automatically by url, now in mmengine define as follows:
'''
register_backend('local', LocalBackend, prefixes='')
register_backend('memcached', MemcachedBackend)
register_backend('lmdb', LmdbBackend)
register_backend('petrel', PetrelBackend, prefixes=['petrel', 's3'])
register_backend('http', HTTPBackend, prefixes=['http', 'https'])
’‘’
So, in this config, you can not use backend_args
of course, you can use like backend_args={'backend': 'petrel'}